File: cpp/program/Keyboard.cpp
#pragma once
#include <iostream>
#include <string>
#include "Peripheral.cpp"

using namespace std;

class Keyboard : public Peripheral{
private:
    string jenisKeyboard;
    string layout;
public:
    Keyboard(){};
    
    Keyboard(string jenisKeyboard, string layout, string merk, string jenis) : Peripheral(merk, jenis){
        this->jenisKeyboard = jenisKeyboard;
        this->layout = layout;
    }
    
    void setjenisKeyboard(string jenisKeyboard){
        this->jenisKeyboard = jenisKeyboard;
    }
    void setlayout(string layout){
        this->layout = layout;
    }

    string getjenisKeyboard(){
        return this->jenisKeyboard;
    }
    string getlayout(){
        return this->layout;
    }
    
    ~Keyboard(){};
};

File: python/program/Keyboard.py
from Peripheral import Peripheral

class Keyboard(Peripheral):
    def __init__(self, jenis_keyboard='', layout='', merk='', jenis=''):
        super().__init__(merk, jenis)
        self.jenis_keyboard = jenis_keyboard
        self.layout = layout

    def set_jenis_keyboard(self, jenis_keyboard):
        self.jenis_keyboard = jenis_keyboard

    def set_layout(self, layout):
        self.layout = layout

    def get_jenis_keyboard(self):
        return self.jenis_keyboard

    def get_layout(self):
        return self.layout